src/System.Private.TypeLoader/src/Internal/Runtime/TypeLoader/EETypeCreator.cs

```diff
@@ -131,6 +131,19 @@ public static void FreeMemory(IntPtr memoryPtrToFree)
         {
             PInvokeMarshal.MemFree(memoryPtrToFree);
         }
+
+        public static unsafe void Copy(IntPtr source, IntPtr destination, int length)
```

Comment:
This does not look used anywhere. Delete it?

(Also, it would be better to use Span.CopyTo instead.)